### PR TITLE
Convert tests to Jest

### PR DESCRIPTION
**Pull request by rylanc**

### CODE REVIEW

File: src/AlwaysMaskedField.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ class AlwaysMaskedField extends React.Component<AlwaysMaskedFieldProps, MaskedFi
   }
 
   private setSelection(start: number, end = start) {
-    if (this.input === document.activeElement) {
+    if (this.input && this.input === document.activeElement) {
```
Comment (colinharris05) [inline src/AlwaysMaskedField.tsx]: What was this change for?

Comment (rylanc) [reply src/AlwaysMaskedField.tsx]: Some change to the types caused it to (rightly) believe that `this.input` can sometimes be `null`, so I have to check it before using it. In reality, it probably never will be, but it won't compile without this check.

**Comment by zp-alfred** [comment]
Accepted by colinharris05!

File: src/AlwaysMaskedField.tsx (hunk 1 above)
Comment (colinharris05) [reply src/AlwaysMaskedField.tsx]: Oooh interesting, cool thanks for explaining

**Pull request merged**